### PR TITLE
XSOAR CI CD playbook to support GitLab

### PR DESCRIPTION
**Pull request by ilaner**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/43797

## Description
Adding support for gitlab in the configuration setup playbook

## Minimum version of Cortex XSOAR
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ContentManagement/ReleaseNotes/1_0_9.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+#### Incident Fields
+- **Configuration File Source**: added option to choose Gitlab as source
```
Comment (ChanochShayner) [inline Packs/ContentManagement/ReleaseNotes/1_0_9.md:3]: Add rn to the PB as well.

Comment (ChanochShayner) [reply Packs/ContentManagement/ReleaseNotes/1_0_9.md:3]:
```suggestion
- **Configuration File Source** 
Added the *Gitlab* option to the single select options.
```

**Review by ChanochShayner — COMMENTED** [review]
Nice work!

File: Packs/ContentManagement/ReleaseNotes/1_0_9.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+#### Incident Fields
+- **Configuration File Source** 
+Added the *Gitlab* option to the single select options.
+
+#### Playbooks
+##### Configuration Setup
+- Getting file from gitlab if gitlab is selected
```
Comment (DeanArbel) [inline Packs/ContentManagement/ReleaseNotes/1_0_9.md:8]:
```suggestion
- Added the option of getting a file from GitLab if **Gitlab** is selected.
```

File: Packs/DevSecOps/ReleaseNotes/1_0_5.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### GitLab
+- Added the **create-file-from-content** argument to the **gitlab-raw-file-get** command, Whether to create a file entry in the War Room with the file contents.
```
Comment (DeanArbel) [inline Packs/DevSecOps/ReleaseNotes/1_0_5.md:4]:
```suggestion
- Added the **create-file-from-content** argument to the **gitlab-raw-file-get** command.
```

**Review by ChanochShayner — APPROVED** [review]

**Pull request merged**